optuna/integration/lightgbm_tuner/optimize.py

```diff
@@ -559,8 +580,10 @@ def tune_feature_fraction(self, n_trials=7):
         # type: (int) -> None
 
         param_name = "feature_fraction"
-        param_values = list(np.linspace(0.4, 1.0, n_trials))
-        sampler = _GridSamplerUniform1D(param_name, param_values)
+        param_values = np.linspace(0.4, 1.0, n_trials).tolist()
+        with warnings.catch_warnings():
```

Comment:
Let's leave a TODO comment to remove this context.

Reply:
Thanks! I added it.